test/unit/units/validateBody.test.js

```diff
@@ -395,12 +435,43 @@ describe('validateBody', () => {
               assert.propertyVal(
                 result.errors[0],
                 'message',
-                `At '/doe' Missing required property: doe`
+                `At '/firstName' Missing required property: firstName`
               );
             });
           });
         });
       });
     });
   });
+
+  // describe.only('when validating Json Schema V3 (Amanda)', () => {
```

Comment:
![dramatic chipmunk](https://proxy.duckduckgo.com/iu/?u=http%3A%2F%2F25.media.tumblr.com%2Ftumblr_lpy1y0aALP1qii6tmo1_250.gif&f=1)

Reply:
Removed during rebase of the existing commit.